kubernetes/client/ws_client.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, configuration, url, headers):
         if headers and 'authorization' in headers:
             header.append("authorization: %s" % headers['authorization'])
 
+        header.append("Sec-WebSocket-Protocol: %s" %
```

Comment:
you should set `ws_streaming_protocol` to `None` in `configuration` object (with a comment explaining what it is) and then here, check if it is not `None`, set the header.